.github/workflows/get-version.yml

```diff
@@ -23,7 +23,7 @@ on:
 
 env:
   JAVA_DIST: 'zulu'
```

Comment:
out of scope, but shouldn't we use the same JDK vendor for all our workflows?

Reply:
We should use the same JDK for all flows building release artifacts.

I think it is not necessary to stick to this rule for all workflows.